lib/components/divider/divider.dart

```diff
@@ -29,7 +28,7 @@ class TUIDivider extends StatelessWidget {
   const TUIDivider({
     super.key,
     this.type = TUIDividerType.horizontal,
-    this.color = TUIDefaultColors.surfaceVariantHover,
+    this.color,
```

Comment:
@kalpeshp0310 Removing the default color i.e. surfaceVariantHover has a visual side-effect. 
https://share.zight.com/4guW651G

Reply:
Ahh, nice catch. Thanks 👍